Makefile

```diff
@@ -251,11 +281,15 @@ test_firefox_standalone_debug:
 	ci \
 	firefox \
 	firefox_debug \
+	opera \
+	phantomjs \
 	generate_all \
 	generate_hub \
 	generate_nodebase \
 	generate_chrome \
 	generate_firefox \
+	generate_opera \
+	generate_phantomjs \
```

Comment:
Why is `node-phantomjs` here? This image was deprecated a while ago.

Reply:
Deleted.